metrics.proto

```diff
@@ -0,0 +1,83 @@
+// FROM https://github.com/prometheus/client_model/blob/proto3/metrics.proto
```

Comment:
good question. i'm not sure, i [requested feedback here](https://github.com/prometheus/client_model/issues/13#issuecomment-302881167).